bin/init.bash

```diff
@@ -82,6 +82,15 @@ generate_base_sc_config() {
     if [[ -f $file ]]; then
         yq merge "$tmpfile" "$file" --inplace -a=overwrite --overwrite --prettyPrint
     fi
+
+    cloud_provider=$(yq r -e "$tmpfile" 'global.cloudProvider')
+    if [ "$cloud_provider" == "exoscale" ]
+    then
+      NFS_SC_SERVER_IP=$( jq -r '.workload_cluster.nfs_ip_addresses.nfs.private_ip' < "${config[infrastructure_file]}")
+      export  NFS_SC_SERVER_IP
```

Comment:
Does this environment variable have to be exported, if we have it in the helmfile values now? :slightly_smiling_face: 
Same for where the wc config is set up.

Reply:
> Does this environment variable have to be exported, if we have it in the helmfile values now? 
> Same for where the wc config is set up.

There is a reference for the environment variables in `bootstrap/storagclass/helmfile/helmfile.yaml ` as shown below. I couldn't find a better way to it.  
```
{{ if eq .Environment.Name "workload_cluster" }}
      server: {{ requiredEnv "NFS_WC_SERVER_IP" }}
{{ end }}
```

Reply:
Would it be possible to remove the reference in the helmfile by setting that value (`nfs.server`) in the `bootstrap/storageclass/helmfile/values/nfs-client-provisioner.yaml.gotmpl` file instead?

```
# nfs-client-provisioner.yaml.gotmpl
...
nfs:
  path: "/nfs"
  server: "{{ .Values.global.nfs_server_ip }}"
```